packages/editor/src/components/post-preview-button/index.js

```diff
@@ -138,59 +94,123 @@ export class PostPreviewButton extends Component {
 
 		// Open up a Preview tab if needed. This is where we'll show the preview.
 		if ( ! this.previewWindow || this.previewWindow.closed ) {
-			this.previewWindow = window.open( '', this.getWindowTarget() );
+			this.previewWindow = window.open( '', '_blank' );
```

Comment:
This is a funny one. I found that using the window name returned from `getWindowTarget` caused `window.open` to focus the iframe in the modal, rather than opening a new window (this although the iframe doesn't have any name set). Switching to `_blank` solves that problem and if a preview tab is already open it will still be reused, so doesn't seem to make any difference.